README.md

```diff
@@ -122,7 +123,14 @@ pip install apache-airflow[postgres,google]==1.10.12 \
  --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-1.10.12/constraints-3.7.txt"
 ```
 
+## Official source code
+
+There is only one "official" source code repository for Apache Airflow, like all ASF projects: https://downloads.apache.org. Official releases are cryptographically signed by the release manager and placed in the Apache svn next to the file, its checksum and archived by ASF in case of any legal disputes.
```

Comment:
Typo around `...its checksum and archived by ASF...`?

Reply:
Thanks - fixed :)